app/controllers/admin/overview_controller.rb

```diff
@@ -4,6 +4,28 @@ class OverviewController < Admin::ApplicationController
     def index
       @open_abuse_reports_count =
         FeedbackMessage.open_abuse_reports.size
+
+      # Stolen from feedback_messages_controller.rb and should probably be adjusted...
```

Comment:
This is still quite a lot of code for a controller, maybe we need one or two more extra services (or query objects, we have of those too) /cc @benhalpern since I think you're helping @ludwiczakpawel pushing this over the finish line technically.

Reply:
👍